src/components/structures/MatrixChat.js

```diff
@@ -191,10 +191,18 @@ module.exports = React.createClass({
             MatrixClientPeg.opts.initialSyncLimit = this.props.config.sync_timeline_limit;
         }
 
+        // Persist the team token across refreshes
+        if (this.props.startingFragmentQueryParams.team_token) {
+            window.sessionStorage.setItem(
```

Comment:
Might be good to comment that yes, we do mean session storage here, for when I completely forget the conversation we had :p